src/shared/bthdef.rs

```diff
@@ -0,0 +1,1127 @@
+// Copyright © 2018 winapi-rs developers
```

Comment:
The copyright lines were removed from `winapi` recently and should be removed from this PR.